### ISSUE TITLE
QT 4.8.7 breaks QGIS GUI

### ISSUE THREAD

**Issue by rbanick** [OSGeo/homebrew-osgeo4mac#95]
Updating from QT 4.8.6 to 4.8.7 recently broke my QGIS GUI. QGIS worked perfectly fine but the majority of the buttons were missing images, rendering it largely impractical to use without memorization. See screenshot below.

![qgis_without_buttons](https://cloud.githubusercontent.com/assets/1583376/8132613/e9edc87c-10f0-11e5-92dc-89f2bb92a336.png)

After some sleuthing on my end the latest QT update was identified as the culprit. I uninstalled qt, manually replaced the 4.8.7 qt.rb file in **user/local/Cellar/Formula** with the 4.8.6 qt.rb file (obtained from a friend's computer), and reinstalled qt. This resolved the problem. To double check I reinstalled 4.8.7 and reopened QGIS -- it was broken again. So this is definitely the issue.

I'm running Mac OS X 10.10.3 and the freshest osgeo4mac installs.

**Comment by geowurster** [comment]
I just discovered the same issue on 10.10.2 and QGIS 2.6 but reverted with:

```console
$ brew uninstall qt
$ brew install https://raw.githubusercontent.com/Homebrew/homebrew/8dd3478131b232553f4c0cab197f8443a0a559a6/Library/Formula/qt.rb
```

**Comment by eegorov** [comment]
Are you using qtcurve?

**Comment by sanak** [comment]
I encountered the same problem.
After building qgis from source, there seems to be no problem,
so I guess that the main cause is the current bottle (https://github.com/OSGeo/homebrew-osgeo4mac/blob/master/Formula/qgis-28.rb#L19).
<pre>
brew uninstall qgis
brew install qgis --build-from-source
</pre>

**Comment by garrettdashnelson** [comment]
Same issue using QGIS 2.8, QT 4.8.7, OS X 10.10.4. I followed @sanak's advice to build from source and it solved the issue.

**Comment by gregmacfarlane** [comment]
The building from source by @sanak resolved for me as well.

**Comment by dakcarto** [comment]
Thanks! Fixed in d0d87f5. Bottles to be added shortly.
